include/glow/Backend/Backend.h

```diff
@@ -119,7 +119,11 @@ class Backend : public Named {
 
   /// \returns true if Constants must be actually quantized before
   /// Post-Lowering, false if it must be done after post-lowering.
-  virtual bool shouldPreQuantizeConstants() const { return true; }
+  /// Prequantize constants flag should only be true when external
+  /// quantization is disabled
```

Comment:
nit: period.

Reply:
@jfix71 I agree with you. It should be something like _CompatibilityDequantization_.